clr-k8s-examples/create_stack.sh

```diff
@@ -97,7 +103,27 @@ function cni() {
 	curl -o "${CANAL_DIR}/overlays/${CANAL_VER}/canal/rbac.yaml" "$CANAL_URL/rbac.yaml"
 	# canal doesnt pass kustomize validation
 	kubectl apply -k "${CANAL_DIR}/overlays/${CANAL_VER}" --validate=false
-
+	;;
+		cilium)
```

Comment:
Make sure shfmt and shellcheck are happy

https://github.com/mvdan/sh
https://github.com/koalaman/shellcheck

Reply:
Done

Reply:
I still see changes when i run `shfmt -w -l create_stack.sh`

Reply:
Did all the shfmt output get fixed?

Reply:
Done